### ISSUE TITLE
pkg(scope): Unable to remove Samsung Game Optimizing Service

### ISSUE THREAD

**Issue by Marekk2k** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#513]
### Describe the problem

It is not possible to fully Samsung Game Optimization Service. After clicking the "Uninstall" button, it is removed for a while, but then it comes back again after refreshing the list in UAD. 

My device: Galaxy A54 5G
Android Version: 14
![Screenshot_20240623_162512_Package Names](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/169854021/e8154605-70c4-4628-9223-9c7a6c44d660)

### Provide logs

[UAD_20240623.log](https://github.com/user-attachments/files/15945027/UAD_20240623.log)

### Acknowledgements

- [X] This issue is not a duplicate of an existing one.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Rudxain** [comment]
[Cross-repo dupe](https://github.com/0x192/universal-android-debloater/issues/193)

**Comment by Rudxain** [comment]
For the record, I want to add that the pack is non-removable from my SM-A315G (Android 12.0, One UI 4.1)

**Comment by AnonymousWP** [comment]
What if you try disabling? Probably some other package is responsible for re-installing it. It likely monitors other apps such as this one, whether they exist or not.

**Comment by Nono-Song** [comment]
I found this [forum](https://xdaforums.com/t/impossible-to-debloat-uninstall-game-optimizing-service.4061417/) discussion about this issue and possible reason behind the scene.
Anyway, I tried to disable instead of uninstalling, and it seems to work at a glance…

**Comment by AnonymousWP** [comment]
> Add the suggestion to the GOS pack description. This is within-scope and trivial to implement, but it's "manual" for the users.

I agree, we could add that you can only disable it. I'll assign this one to my name. It doesn't require manual intervention; users just need to use the disable mode in UAD-ng.
